catalog/pipeline/deploy-release/0.1/deploy-release.yaml

```diff
@@ -0,0 +1,34 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: deploy-release-pipelinerun
```

Comment:
updated name to be `deploy-release-pipeline`

As for the pipeline content, you are right it only contains a `ec-validate call`.
However, it may sometime soon require additional tasks, such as signing,  to be added.

I'd rather keep the name inline with the intended strategy.

Reply:
I personally think we should do our pipelines as we do our tasks and have the `metadata.name` be the directory name.
If/when additional tasks are added, I'd hope that the description of this pipeline changes with the addition

Reply:
so `metadata.name = deploy-release`?

Reply:
updated

Reply:
yes, that would be my vote